### DISCUSSION TITLE
I can't get chia from the exchange

### DISCUSSION THREAD

**Discussion by Zev87** [Chia-Network/chia-blockchain#17182] [Support]
Hello, I have a problem, I can't get a withdrawal 3 XCH from the exchange to my official chia wallet. I contacted exchange support twice and can't get a positive result from them. Their support directs me to contact support of the Chia network for information about the confirmation of the block and that the funds are no longer in exchange accounts.
Update: I have made some progress in communicating with the exchange's support. I will update and supplement the information in case the problem is not resolved

**Comment by BrandtH22** [comment]
Hey @Zev87 , thank you for the update.
As a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia). 

I am going to close this ticket for the time being but do not hesitate to reopen it or join us in our discord if any questions arise or assistance from our side is needed :+1:
